test/ipc_test.js

```diff
@@ -229,3 +229,41 @@ it('#6 GetPTTRequest *with* addresses - mock server', async function() {
         });
   });
 });
+
+it('#7 GetTips - mock server', async function() {
```

Comment:
would be great if you'll be able to explain to me what this test does exactly :)